src/@utils/subgraph.ts

```diff
@@ -5,6 +5,7 @@ import { getOceanConfig } from './ocean'
 import { AssetPreviousOrder } from '../@types/subgraph/AssetPreviousOrder'
 import { OrdersData_orders as OrdersData } from '../@types/subgraph/OrdersData'
 import { OpcFeesQuery as OpcFeesData } from '../@types/subgraph/OpcFeesQuery'
+import { chainIdsSupported } from 'app.config'
```

Comment:
to keep imports constant through the app we should use relative paths here also
```suggestion
import appConfig from '../../app.config'
```